requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
Apparently we can get this working under tox instead: https://tox.readthedocs.org/en/latest/example/general.html